README.md

```diff
@@ -90,14 +91,17 @@
 
 
 
-# 👥 Authors
+# 👥 Author
 
 ### Adesina Abiodun Paul
 
-# 👤 Author1
+
 
 ### GitHub: https://github.com/adesina-abiodun
 
+### LinkedIn: https://www.linkedin.com/in/adesina-abiodun-paul-b513bb143
+### Twitter: https://twitter.com/adesinaabiodunp
```

Comment:
The link to your `LinkedIn` profile is currently broken, kindly fix that. 💪

![image](https://user-images.githubusercontent.com/78251927/234798789-230a625b-fac5-4a83-96bb-4e092b79eb8a.png)

Let's avoid broken links in our readme file for a professional repository. 👍 🤗